### PR TITLE
Feature/#1783 mute others in meeting

### PR DESCRIPTION
**Pull request by aminbksi**

### CODE REVIEW

File: src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx [hunk 1]
```diff
@@ -164,6 +163,24 @@ const CommunicationLayer: React.FC<CommunicationLayerProps> = () => {
...
+  useWebsocketEvent('meeting-mute', () => {
+    collaborationDispatch({
+      type: COLLABORATION_MUTED_ACTION_UPDATE,
+      muted: true
+    });
+  });
+
+  useWebsocketEvent('meeting-mute-all', (moderatorId) => {
+    if (currentUserId === moderatorId) {
+      return;
```
Comment (Danchev1) [inline src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: cant we just put the collaborationDispatch inside if and reverse check

Comment (aminbksi) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: I want not to mute the moderator who muted all, why do you thing it is better to have a reverse check?

Comment (Danchev1) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: if return; next lines are redundant

Comment (Danchev1) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: at least that is what i see from a glance

Comment (aminbksi) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: it's only redundant when the statement is true. only once

Comment (Danchev1) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx:175]: yes

**Review by gerbenrobijn-sigbar — APPROVED** [review]

File: src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx (hunk 1 above)
Comment (aminbksi) [reply src/scenes/communication/pages/CommunicationLayer/CommunicationLayer.tsx]: done

**Review by Danchev1 — APPROVED** [review]

**Pull request merged**